### ISSUE TITLE
Experiment: 2x Longer Games + High Learning Rate + Combined Best Settings

### ISSUE THREAD

**Issue by LiamConnell** [LiamConnell/FootyAI#25]
## Summary
Test extended gameplay duration (2x longer episodes) to allow more complex strategic development while maintaining aggressive learning settings.

## Configuration
- **GAME_DURATION = 180** (2x baseline of 90 time steps)
- LEARNING_RATE = 5e-4 (5x baseline)
- SIGMA_MULTIPLIER = 1.0 (proven exploration)
- BATCH_SIZE = 512 (maximum sample efficiency)
- ENABLE_KICK_REWARD = False (cleaner signal)
- ENABLE_DISTANCE_REWARD = True (ball proximity guidance)
- REWARD_DECAY_EPISODES = 5000 (curriculum learning)

## Hypothesis
Longer episodes should enable:
1. **More complex strategies** - agents have more time to develop sophisticated plays
2. **Better coordination** - teammates can execute multi-step collaborative moves
3. **Richer learning signal** - more diverse situations per episode
4. **Strategic depth** - time for setup plays, positioning, counter-attacks

**Trade-off:** Longer episodes may slow training (fewer episodes per hour) but should improve strategy quality.

## Expected Outcomes
- **Higher quality strategies** - more realistic soccer gameplay
- **Better team coordination** - agents learn to work together over longer sequences
- **More diverse behaviors** - larger action space exploration within episodes
- **Slower episode throughput** - but potentially better sample efficiency per episode

## Key Questions
1. Do longer episodes lead to qualitatively better soccer strategies?
2. Does the extended time horizon improve team coordination?
3. Is the trade-off of fewer episodes worth the strategic complexity?

## Comparison Baselines
- Issue #24: Same config with GAME_DURATION=90 (1x baseline)
- All other experiments: 90 time step episodes

## Deployment Command
```bash
# Update game duration and deploy
uv run python deploy_vertex_ai.py
```

## Status
- [ ] Update GAME_DURATION to 180
- [ ] Deploy to Vertex AI
- [ ] Monitor training progress and episode quality
- [ ] Compare strategy complexity vs shorter episodes
- [ ] Analyze team coordination improvements

## Related
Builds on high learning rate experiment (Issue #24), testing episode length impact on strategy development.
